dev-java/java-service-wrapper/files/java-service-wrapper-3.5.25-gentoo-wrapper-defaults.patch

```diff
@@ -0,0 +1,25 @@
+|Original version of this patch by Ralph Sennhauser <sera@gentoo.org>
```

Comment:
I'd include a reference to the Gentoo bug at the top here, given we know it's still needed atm.

Reply:
> I'd include a reference to the Gentoo bug at the top here, given we know it's still needed atm.

Referring to https://bugs.gentoo.org/878869#c3 saying »The main service the wrapper provides is auto-update over Freenet, and that is not useful for freenet installed via ebuild.« I'm not sure it's really needed.
In https://github.com/gentoo/gentoo/pull/30643 I've dropped it from runtime classpath to compile classpath.